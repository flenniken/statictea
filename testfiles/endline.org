# $ block name = "endline"
# $ : t.repeat = 2
# $ : description = "looking for extra newlines"
** {name}()
:PROPERTIES:
:CUSTOM_ID: {name}
:END:

{description}
# $ endblock
------------

