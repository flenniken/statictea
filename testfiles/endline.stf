id stf file version 0.0.0

# Test readmefun ending line case.

--- file cmd.sh command
$statictearoot/bin/statictea -p="# $" -t=endline.org >stdout 2>stderr
--- endfile

--- file endline.org
# $ block name = "endline"
# $ : t.repeat = 2
# $ : description = "looking for extra newlines"
** {name}()
:PROPERTIES:
:CUSTOM_ID: {name}
:END:

{description}
# $ endblock
------------
endfile

--- file stdout.expected
** endline()
:PROPERTIES:
:CUSTOM_ID: endline
:END:

looking for extra newlines
** endline()
:PROPERTIES:
:CUSTOM_ID: endline
:END:

looking for extra newlines
------------
endfile

--- file stderr.expected
--- endfile

--- expected stdout.expected == stdout
--- expected stderr.expected == stderr

