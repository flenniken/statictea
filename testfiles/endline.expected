** endline()
:PROPERTIES:
:CUSTOM_ID: endline
:END:

looking for extra newlines
** endline()
:PROPERTIES:
:CUSTOM_ID: endline
:END:

looking for extra newlines
------------

